proto/sharedstreets.proto

```diff
@@ -28,6 +28,11 @@ enum RoadClass {
   Other = 8;
 }
 
+message OsmTag {
```

Comment:
Should this be tied to OSM? I'm wondering if the same concept of a metadata blob could apply to other datasets just as effectively.